common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -290,9 +294,19 @@ def test_deferred_verification_enrollment(self):
         enrollment_mode = self.dashboard_page.get_enrollment_mode(self.course_info["display_name"])
         self.assertEqual(enrollment_mode, 'verified')
 
+        # Expect enrollment activated event
+        self.assertEqual(
```

Comment:
This is awesome, could we move this logic into some kind of helper so we can use it more pervasively in these tests?

I'd like to be able to just put into any test without the boilerplate (time checking, mongo client setup etc).

```python
self.assertExactlyOneEventEmitted({
    "name": "edx.course.enrollment.activated",
    "event.user_id": int(student_id)
})
```